zebrad/src/commands/start.rs

```diff
@@ -28,7 +28,9 @@ use color_eyre::eyre::{eyre, Report};
 use futures::{select, FutureExt};
 use tokio::sync::oneshot;
 use tower::builder::ServiceBuilder;
+use tower::util::BoxService;
 
+use crate::components::mempool::Mempool;
```

Comment:
No biggie but i think we can merge all the `use crate::` in 1 statement with this one and the ones in the next 6 lines.

Reply:
Thanks, I always forget to double-check how the imports look. Done in 889c9549